### PR TITLE
done with the buttons component

### PR DESCRIPTION
**Pull request by sohan9819**
done with the buttons components and added code snippets to docs

### CODE REVIEW

File: alpha/docs.html [hunk 1]
```diff
@@ -759,6 +759,210 @@ <h2 class="badge__basic-content-m h2">TEXT</h2>
...
+            <a class="component--title h2"># Buttons</a>
+
+            <!-- ##### COMPONENT BLOCK : BUTTONS BASIC TYPES #### -->
+
+            <div class="alert__demo-block m-1">
+              <h3 class="button__demo-title h3"># Button Types</h3>
+              <div class="button__demo-buttons flex-center">
+                <a
+                  href="#"
+                  class="uppercase h4 btn--medium border--rad-s btn__basic__alpha-text"
```
Comment (kirti2602) [inline alpha/docs.html:775]: `btn--medium border--rad-s` this class name is way too long and also not very clear what `--rad-s` means. So maybe try shorter and self-descriptive class names

File: css/components/buttons.css [hunk 2]
```diff
@@ -14,3 +14,140 @@
...
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.btn__icon-l:link,
+.btn__icon-l:visited {
+  width: 6rem;
+  height: 6rem;
+  padding: 2rem;
+  display: flex;
```
Comment (kirti2602) [inline css/components/buttons.css:141]: one suggestion here, you can try making a utility class out of `display: flex` and reuse that class whenever necessary.

File: css/components/buttons.css [hunk 3]
```diff
@@ -14,3 +14,140 @@
...
+  justify-content: center;
+  align-items: center;
+}
+.btn__icon-xl:link,
+.btn__icon-xl:visited {
+  width: 10rem;
+  height: 10rem;
+  padding: 2rem;
+  display: flex;
```
Comment (kirti2602) [inline css/components/buttons.css:150]: here as well. can be put into utility class to avoid repetition of property

**Review by kirti2602 — APPROVED** [review]
Can work on suggested changes. Rest looks fine to me. Good work

File: alpha/docs.html (hunk 1 above)
Comment (sohan9819) [reply alpha/docs.html:775]: That class means border radius small . I thought of making it small but did got any good name . Should I make it just "rounded-s" 😅

File: css/components/buttons.css (hunk 2 above)
Comment (sohan9819) [reply css/components/buttons.css:141]: Yeah working on it 🐱

**Comment by krgarima** [comment]
dark2
This is how your site looks in chrome. In Firefox, it looked good. Work on browser compatibility once done with all the components.

**Comment by krgarima** [comment]
dark1
Are you following BEM here? Use double dash (--) when using BEM convention instead of single -

File: alpha/docs.html (hunk 1 above)
Comment (kirti2602) [reply alpha/docs.html:775]: > That class means border radius small . I thought of making it small but did got any good name . Should I make it just "rounded-s" 😅

What about round-sm or rounded-sm? You can pick any depending on your choice

Comment (sohan9819) [reply alpha/docs.html:775]: ok I will do so 👍

**Comment by sohan9819** [comment]
@krgarima sorry I didn't get it ,   I guess I am already using -- where you are saying me to use . And  thanks for pointing about browser compatibility , I will sure work on it at the end .

**Comment by krgarima** [comment]
> @krgarima sorry I didn't get it , I guess I am already using -- where you are saying me to use . And thanks for pointing about browser compatibility , I will sure work on it at the end .

It should be btn__basic--text.

**Comment by sohan9819** [comment]
> 

I used a single dash because "text" is one of the sub options of btn__basic . But I  have made some mistakes in naming the classes which I thought of fixing in the dev branch directly just to avoid conflicts .

**Review by Rutwik87 — APPROVED** [review]
Good to go @sohan9819

File: css/components/buttons.css (hunk 2 above)
Comment (sohan9819) [reply css/components/buttons.css:143]: done! added utility for flex . Thanks for the suggestion .

**Pull request merged**